test/OrchardCore.Tests/Apis/GraphQL/Queries/RecentBlogPostsQueryTests.cs

```diff
@@ -5,7 +5,7 @@
 using OrchardCore.Tests.Apis.Context;
 using Xunit;
 
-namespace OrchardCore.Tests.Apis.GraphQL.Queries
+namespace OrchardCore.Tests.Apis.GraphQL
```

Comment:
Why do you change the namespace?

Reply:
for better top level grouping of the tests in explorer ;)

eg ..

![image](https://user-images.githubusercontent.com/465424/69811427-f2fbe980-11e5-11ea-95e9-9f8f4e2063a4.png)